app/presenters/finder_breadcrumbs_presenter.rb

```diff
@@ -0,0 +1,23 @@
+class FinderBreadcrumbsPresenter
+  def initialize(parent_content_item, finder_content_item)
+    @parent_content_item = parent_content_item
+    @finder_name = finder_content_item.dig("title")
+  end
+
+  def breadcrumbs
+    crumbs = [{ title: "Home", url: "/" }]
+
+    if @parent_content_item.dig("document_type") == "organisation"
+      crumbs << { title: 'Organisations', url: '/government/organisations' }
+      unless @parent_content_item.dig("title").nil? || @parent_content_item["title"].empty?
```

Comment:
You can simplify this line by using `if @parent_content_item.dig("title").present?` I think:
> irb(main):002:0> nil.present?
> => false
> irb(main):003:0> "".present?
> => false

Reply:
Amazing, we were trying to find a more succinct way of doing this